configs/task_twip.yaml

```diff
@@ -4,8 +4,14 @@ policy: "MlpPolicy"
 seed: 14321
 
 n_envs: 128
-timesteps_per_env: 1_000_000
-base_lr: 1e-4
+timesteps_per_env: 500_000
+base_lr: 3e-4
+
+delay:
+  enabled: True
+  obs_delay_range: [1, 8] # sample range of time steps to delay the observation (same for action)
+  act_delay_range: [1, 2]
```

Comment:
Comment a bit confusing is the delay_range for action same as ob([1,8]) or is it just [1,2]

Reply:
I'll clarify!